bindings/wasm/Cargo.toml

```diff
@@ -28,6 +28,10 @@ serde_repr = { version = "0.1", default-features = false }
 wasm-bindgen = { version = "0.2.82", features = ["serde-serialize"] }
 wasm-bindgen-futures = { version = "0.4", default-features = false }
 
+# Pin tokio version as a orkaround for compliation error caused by `iota-client`.
+# TODO unpin version once issue is resolved.
+tokio = { version = "=1.20.1", default-features = false }
```

Comment:
The issue with `iota-client`has been fixed on `dev` now so this is no longer needed.